package.json

```diff
@@ -60,6 +61,8 @@
   },
   "dependencies": {
     "@kubernetes/client-node": "^0.11.1",
+    "@material-ui/core": "^4.9.10",
```

Comment:
Please, move react back to dev dependencies.

Reply:
Fixed in https://github.com/redhat-developer/vscode-openshift-tools/pull/1520/commits/028b996abb117c745a9e03d01d34f1e277542832